src/pages/BrowseSchemes.svelte

```diff
@@ -162,10 +163,11 @@
               <RailwayStationsLayerControl />
             </CheckboxGroup>
           </CollapsibleCard>
-          <CollapsibleCard label="Road network">
+          <CollapsibleCard label="Existing infrastructure">
```

Comment:
Renamed this for the moment, I still expect churn as we add more and recategorize